Makefile

```diff
@@ -1,19 +1,22 @@
 SRC := src
 OBJ := obj
 
-FLAGS := -Wall -Wpedantic -Wextra
```

Comment:
Debug flags should not be the default, maybe add them in a custom `debug` target?